imports/minutes.js

```diff
@@ -415,6 +415,16 @@ export class Minutes {
 
         return this.participants;
     }
+    /**
+     * Change presence of a all participants in a Minute
+     * @param isPresent new state of presence
+     */
+    changeParticipantsStatus(isPresent)
+    {
+        for (var index = 0; index < this.participants.length; index++){
```

Comment:
This for-loop calls implicitly a meteor method in each iteration. There are two drawbacks with this approach: a lot of network traffic and the method `updateParticipantPresent` returns a promise which will be ignored so the user will not be notified if any error occurs.

Have a look what the method `updateParticipantPresent` does: it changes the state of the participant with the given index and updates the complete `participants` array of the minutes document. This means each call sends the complete list with all participants to the server.

I would prefer changing each participant first and then update the document in one update-call. By returning the result of `this.update(...)` you pass the returning promise back to the caller. When calling the `changeParticipantsStatus` from within the template click-listener you can simply catch any error which might be occur and pass it to our `handleError` function like we are doing it in other templates.